explainer.md

```diff
@@ -690,13 +706,16 @@ enum VRFrameOfReferenceType {
 
 interface VRFrameOfReference : VRCoordinateSystem {
   readonly attribute VRStageBounds? bounds;
+  attribute EventHandler onboundschange;
```

Comment:
Do we want to pass in the update bounds as part of the event?  If so, doesn't the event need to be a VRFrameOfReferenceEvent rather than a VRCoordinateSystemEvent?